debian/control

```diff
@@ -35,9 +35,9 @@ Depends: ${shlibs:Depends},
          sssd-dbus,
          apparmor,
          cifs-utils,
-         curlftpfs,
 Recommends: ${misc:Recommends},
             ubuntu-advantage-desktop-daemon,
+            curlftpfs,
```

Comment:
This is a Recommends, not a Suggests. As we discussed duringour HO, Ubuntu installs recommends by default, but not Suggests.

Reply:
Oh, I switched the definitions mentally... My bad, brain is still a bit slow after the holidays